src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1739,11 +1716,8 @@ public class OpcodeWait : Opcode
 
         public override void Execute(ICpu cpu)
         {
-            object waitTime = cpu.PopValue();
-            if (waitTime is double)
-                cpu.StartWait((double)waitTime);
-            else if (waitTime is int)
-                cpu.StartWait((int)waitTime);
```

Comment:
cpu.StartWait actually takes a double anyway, so it seemed ridiculous to cast an int value to an int, pass it to StartWait, which will coerce it to a double.  May as well just convert it to a double unconditionally whether it was a double or an int.